core/store/src/test/resources/META-INF/services/mil.nga.giat.geowave.core.store.spi.CLIOperationProviderSpi

```diff
@@ -0,0 +1 @@
+mil.nga.giat.geowave.core.store.operations.config.addstore.AddStoreOperationProvider
```

Comment:
did we add a new test that requires this?

Reply:
deleted